modal.css

```diff
@@ -0,0 +1,273 @@
+.center,.content{
+    position: absolute;
+    top:50%;
+    left:50%;
+    transform:translate(-50%, -50%);
+}
+
+#click{
+    display:none;
+}
+
+.center .click-me{
+    display: block;
+    width:160px;
+    height:50px;
+    background: var(--secbg-color);
+    border: 1px solid var(--mainBrandColor);
+    color:var(--font-primary);
+    text-align: center;
+    font-size: 22px;
+    line-height:50px;
+    border-radius: 3px;
+    cursor: pointer;
+    transition: 0.5s;
+}
```

Comment:
Hey hi I actually wanted to ask that you are declaring center  again and giving other properties
Suggestion-make some utility classes so it will increase readability of code and also it will  broaden the use case of the classes and reduce the line of code ultimately